test_projects/cnode/bundlex.exs

```diff
@@ -8,13 +8,14 @@ defmodule Example.BundlexProject do
   end
 
   def natives(_platform) do
+    language = System.get_env("UNIFEX_TEST_LANG") |> String.to_atom()
     [
       example: [
         src_base: "example",
-        sources: ["example.c"],
+        sources: ["example.#{language}"],
```

Comment:
Do C and C++ files differ from each other? If not, I'd try not to duplicate their contents

Reply:
There is no difference between these two files, but C++ compiler generates warnings on try of compiling a file with `.c` extension

Reply:
So maybe we could create the cpp files dynamically in test?